Presentation.Web/Models/GDPR/DataProcessingRegistrationDTO.cs

```diff
@@ -15,6 +16,10 @@ public DataProcessingRegistrationDTO(int id, string name)
 
         public NamedEntityWithEnabledStatusDTO[] ItSystems { get; set; }
 
+        public YearMonthIntervalOption? OversightIntervalOption { get; set; }
```

Comment:
Lav en ny DTO type og brug den. Det er ikke sidste gang vi får en værdi hvortil der er tilknyttet en note. Du kan kalde det nye felt på denne dto for OversightInterval:

her er bud på den nye DTO
 
```
class ValueWithNoteDTO<T> {
   public T Value {get;set;}
   public string Note {get;set;}
}
```